### PR TITLE
Use GitHub icon for source code link

### PR DESCRIPTION
**Pull request by simonbaird**
I thought it looked better on the right, so I also moved it.

### CODE REVIEW

File: antora/supplemental-ui/partials/header-content.hbs [hunk 1]
```diff
@@ -66,6 +64,13 @@
 
         --}}
 
+        {{! GitHub logo SVG from https://icons.getbootstrap.com/icons/github/ }}
+        <a class="navbar-item" href="https://github.com/enterprise-contract" title="Source Code">
+          <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="currentColor" class="bi bi-github" viewBox="0 0 16 16">
+            <path d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.012 8.012 0 0 0 16 8c0-4.42-3.58-8-8-8z"/>
```
Comment (lcarva) [inline antora/supplemental-ui/partials/header-content.hbs]: It would be nice if this came from a variable. But I have no idea if that's possible.

**Review by lcarva — APPROVED** [review]

File: antora/supplemental-ui/partials/header-content.hbs (hunk 1 above)
Comment (simonbaird) [reply antora/supplemental-ui/partials/header-content.hbs]: It could be in its own partial probably..

Comment (simonbaird) [reply antora/supplemental-ui/partials/header-content.hbs]: Pushed a new commit which uses a shared file with symlinks in two places. Maybe not the most beautiful solution, but it seems to work okay.

File: antora/package.json [hunk 2]
```diff
@@ -17,7 +17,8 @@
     "glob-parent": "6.0.2"
   },
   "scripts": {
-    "build": "antora generate --clean --fetch antora-playbook.yml"
+    "build": "antora generate --clean --fetch antora-playbook.yml",
+    "fast-build": "antora generate antora-playbook.yml"
```
Comment (zregvart) [inline antora/package.json]: Perhaps

```suggestion
    "build:base": "antora generate antora-playbook.yml",
    "build": "npm run build:base -- --clean --fetch",
    "build:fast": "npm run build:base",
```

Comment (simonbaird) [reply antora/package.json]: Done, thanks.

**Comment by simonbaird** [comment]
Let's merge.

**Pull request merged**